### DISCUSSION TITLE
[@types/gm] imageMagick option of `subClass` should also allow strings

### DISCUSSION THREAD

**Discussion by wiggisser** [DefinitelyTyped/DefinitelyTyped#62703] [Issues with a @types package]
According to the docs of the [gm](https://www.npmjs.com/package/gm) package, for using latest imageMagick instead of graphicsMagick you should do

    const gm = require('gm').subClass({ imageMagick: '7+' });

which would be translated to TS roughly as follows

    import { subClass } from "gm"
    const gm = subClass({ imageMagick: '7+'});

but the current typings in `@types/gm` have defined the `imageMagick` property as  `imageMagick: boolean | undefined`, so you are not allowed to pass a string ...

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62839
